extension/src/Telemetry.ts

```diff
@@ -16,7 +16,8 @@ export function startTelemetry(
   extensionPackage: IExtensionPackage
 ): void {
   if (!initiated) {
-    enableTelemetry = settings.enableTelemetry && !process.env.GITHUB_ACTION;
+    enableTelemetry =
+      settings.enableTelemetry && !process.env.NAB_DISABLE_TELEMETRY;
```

Comment:
There is the posibility to move the env check to `Settings` I guess.

Reply:
nah, leave it like this. 👍